plugins/obs-webrtc/whip-output.cpp

```diff
@@ -1,3 +1,4 @@
+#include <obs.hpp>
 #include "whip-output.h"
 #include "whip-utils.h"
```

Comment:
I think our preferred header order is:
```cpp
#include "whip-output.h"
#include "whip-utils.h"

#include <obs.hpp>
```

Unless there is a specific reason they must be in this order.

Reply:
@RytoEX fixed! Could I get another review, thank you :)

Reply:
Thanks.